### PR TITLE
Add method `all_classes` to allow lazy load on backends class

### PR DESCRIPTION
**Pull request by drgarcia1986**
This method is useful for cases when you need to use some classmethod of a backend without having to create an instance of it

### CODE REVIEW

File: ramos/pool.py [hunk 1]
```diff
@@ -35,11 +35,11 @@ def all(cls, *args, **kwargs):
 
         return [
             backend_class.create(*args, **kwargs)
-            for backend_class in cls._get_backends_classes()
+            for backend_class in cls.all_lazy()
         ]
 
     @classmethod
-    def _get_backends_classes(cls):
+    def all_lazy(cls):
```
Comment (lsmag) [inline ramos/pool.py]: You may consider this a nitpick, but what do you think about naming it `all_classes` or something like that? I understand how "lazy" here refers to "it's not being instantiated", but lazy values have other meanings elsewhere too, which don't apply here.

Technically speaking, it's not returning lazy objects or data, it's returning classes

Comment (renanivo) [inline ramos/pool.py]: What about naming it `iterator()`? [Like that famous framework](https://docs.djangoproject.com/en/2.2/ref/models/querysets/#iterator)

Comment (drgarcia1986) [reply ramos/pool.py]: Wow good point @lsmag you're right

Comment (lsmag) [reply ramos/pool.py]: to me it's a nitpick, feel free to ignore my original comment

Comment (drgarcia1986) [reply ramos/pool.py]: No, it makes sense, I changed

**Comment by drgarcia1986** [comment]
@renanivo good idea 👍

**Comment by drgarcia1986** [comment]
@renanivo updated

**Pull request merged**